diff -Naur a/host/cmake/modules/FindFFTW.cmake b/host/cmake/modules/FindFFTW.cmake
--- a/host/cmake/modules/FindFFTW.cmake	2021-03-20 03:14:30.000000000 +0300
+++ b/host/cmake/modules/FindFFTW.cmake	2021-11-14 21:10:41.269260500 +0300
@@ -12,12 +12,12 @@
 
 find_path (FFTW_INCLUDES fftw3.h)
 
-IF (WIN32)
+IF (FFTW_LIBRARIES)
 include_directories(${FFTW_INCLUDES})
 find_library (FFTW_LIBRARIES NAMES ${FFTW_LIBRARIES})
 ELSE(WIN32)
 find_library (FFTW_LIBRARIES NAMES fftw3)
-ENDIF(WIN32)
+ENDIF(FFTW_LIBRARIES)
 
 
 
diff -Naur a/host/cmake/modules/FindUSB1.cmake b/host/cmake/modules/FindUSB1.cmake
--- a/host/cmake/modules/FindUSB1.cmake	2021-03-20 03:14:30.000000000 +0300
+++ b/host/cmake/modules/FindUSB1.cmake	2021-11-14 21:13:34.829335700 +0300
@@ -30,7 +30,8 @@
   ENDIF(${CMAKE_SYSTEM_NAME} MATCHES "FreeBSD")
 
   FIND_PATH(LIBUSB_INCLUDE_DIR libusb.h
-    PATHS ${PC_LIBUSB_INCLUDEDIR} ${PC_LIBUSB_INCLUDE_DIRS})
+    PATHS ${PC_LIBUSB_INCLUDEDIR} ${PC_LIBUSB_INCLUDE_DIRS}
+    PATH_SUFFIXES libusb-1.0)
 
   FIND_LIBRARY(LIBUSB_LIBRARIES NAMES ${LIBUSB_LIBRARY_NAME}
     PATHS ${PC_LIBUSB_LIBDIR} ${PC_LIBUSB_LIBRARY_DIRS})
@@ -40,4 +41,4 @@
 
   MARK_AS_ADVANCED(LIBUSB_INCLUDE_DIR LIBUSB_LIBRARIES)
 
-endif (LIBUSB_INCLUDE_DIR AND LIBUSB_LIBRARIES)
\ No newline at end of file
+endif (LIBUSB_INCLUDE_DIR AND LIBUSB_LIBRARIES)
